src/views/newConnection.tsx

```diff
@@ -150,11 +185,32 @@ export default class newConnectionTemplates {
 
   public stepper = (params: { formStage: FormStage }): JSX.Element => {
     if (params.formStage === 'form') {
-      return <div>-----Step 1</div>
+      return (
+        <>
```

Comment:
you don't need the fragments here as there is only one child (same with the others in this function), remove